lib/store_attribute/active_record/store.rb

```diff
@@ -263,6 +269,26 @@ def _define_predicate_method(name, prefix: nil, suffix: nil)
           end
         end
       end
+
+      def _ensure_attributes_sync_module_included
+        unless included_modules.include?(StoreAttribute::ActiveRecord::Store::AttributesSync)
+          include StoreAttribute::ActiveRecord::Store::AttributesSync
+        end
+      end
+
+      def _define_store_attribute_accessors(store_name, name)
+        # Override getter to read from store
```

Comment:
Do we need to override the getter? It's already define by the built-in store-accessor.

As I understand, we need to hack the writer to keep things in sync. For that, I would suggest having a dedicated module prepended with the sync code defined and a `super` call instead.

And this module should only be included if the feature is enabled.